packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -90,7 +90,9 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
         let addedAllNodes: boolean = true;
         for (const nodeUri of nodeUris) {
             try {
-                let nodes: FabricNode | Array<FabricNode> = await fs.readJson(nodeUri.fsPath);
+                const nodeContents: Uint8Array = await vscode.workspace.fs.readFile(nodeUri);
+                const nodeString: string = nodeContents.toString();
+                let nodes: FabricNode | Array<FabricNode> = JSON.parse(nodeString);
```

Comment:
Following on from the previous comment we could even have ExtensionUtil.readJSON(path).